### PR TITLE
VRRAP: Create form #22315

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

https://github.com/department-of-veterans-affairs/va.gov-team/issues/22315

## Screenshots
Screen Shot 2021-03-30 at 3 12 58 PM
Screen Shot 2021-03-30 at 3 13 08 PM
Screen Shot 2021-03-30 at 3 13 18 PM
Screen Shot 2021-03-30 at 3 13 28 PM
Screen Shot 2021-03-30 at 3 13 37 PM
Screen Shot 2021-03-30 at 3 18 30 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/1990s/content/directDeposit.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+export const directDepositDescription = () => {
+  return (
+    /* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */
+    <div tabIndex="0">
+      <p>
+        We make payments only through direct deposit, also called electronic
+        funds transfer (EFT). If you’re approved for the Rogers STEM
+        Scholarship, we’ll need to know where to deposit the funds.
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/1990s/content/directDeposit.jsx:11]: We're not applying for the STEM scholarship

File: src/applications/edu-benefits/1990s/content/directDeposit.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  });
+};
+
+export const directDepositAlert = () => (
+  // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+  <div className="vads-u-padding-top--1p5" tabIndex="0">
+    <p>
+      <b>Note:</b> Any bank account information you enter here will apply to
+      your other Veteran benefits, including compensation, pension, and Benefits
+      for Certain Children with Disabilities (Chapter 18) payments.
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/1990s/content/directDeposit.jsx:34]: This note is incorrect

File: src/applications/edu-benefits/1990s/pages/application.js [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+const uiSchema = {
+  'view:applicantInformation': {
+    'ui:title': 'Applicant Information',
+    veteranFullName: fullNameUI,
+    veteranSocialSecurityNumber: ssnUI,
+    dateOfBirth: {
+      ...currentOrPastDateUI('Date of birth'),
+      'ui:errorMessages': {
+        required: 'Please provide a valid date',
+        futureDate: 'Please provide a valid date',
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/1990s/pages/application.js:62]: Should this be something like 'Please provide a valid date in the past'?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/edu-benefits/1990s/pages/application.js (hunk 3 above)
Comment (zurbergram) [reply src/applications/edu-benefits/1990s/pages/application.js:62]: Copy and paste from 0994, will update

File: src/applications/edu-benefits/1990s/content/directDeposit.jsx (hunk 2 above)
Comment (zurbergram) [reply src/applications/edu-benefits/1990s/content/directDeposit.jsx]: Fixed

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/edu-benefits/1990s/content/directDeposit.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/edu-benefits/1990s/content/directDeposit.jsx]: Fixed

**Pull request merged**